sdk/storage/azure_storage_blob/src/clients/blob_client.rs

```diff
@@ -91,16 +100,10 @@ impl BlobClient {
     /// * `options` - Optional configuration for the request.
     pub async fn get_blob_properties(
```

Comment:
This method name doesn't align with the method options type name.

Reply:
True, I'm fine if we want to rename this method to `get_properties` to align with generated code.

Reply:
Sounds good, changed all instances of `get_x_properties` -> `get_properties` to align with generated code.